src/geo/map/tooltip-template.js

```diff
@@ -0,0 +1,27 @@
+var _ = require('underscore');
+var Backbone = require('backbone');
+
+var TooltipTemplate = Backbone.Model.extend({
```

Comment:
I don't see the benefits of using cdb.core.Model over using a good old Backbone Model here.... that's why I always go for Backbone.Model...